cmd/image-inspector.go

```diff
@@ -18,24 +18,26 @@ func main() {
 	flag.StringVar(&inspectorOptions.DstPath, "path", inspectorOptions.DstPath, "Destination path for the image files")
 	flag.StringVar(&inspectorOptions.Serve, "serve", inspectorOptions.Serve, "Host and port where to serve the image with webdav")
 	flag.BoolVar(&inspectorOptions.Chroot, "chroot", inspectorOptions.Chroot, "Change root when serving the image with webdav")
+	flag.BoolVar(&inspectorOptions.Local, "local", inspectorOptions.Local, "If set the image wont be pulled but instead the -path will be used")
```

Comment:
@mfojtik if this is related to scanning (e.g. scan local path) shouldn't contain a scan prefix? e.g. `scan-local-path` or something similar.